companies/views.py

```
@@ -252,6 +293,86 @@ def get_company_reports(request, stock_code):
+                "message": "보고서 분석 결과 조회 성공",
+                "data": serializer.data,
+            },
+            status=status.HTTP_200_OK,
+        )
+
+    except Company.DoesNotExist:
+        return Response(
+            {"status": 404, "error": "Company not found"},
+            status=status.HTTP_404_NOT_FOUND,
+        )
+    except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)